bundles/framework/announcements/view/AnnouncementsContent.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Message } from 'oskari-ui';
+import styled from 'styled-components';
+import { SelectOutlined } from '@ant-design/icons';
+
+const LinkIcon = styled(SelectOutlined)`
+    margin-left: 6px;
+`;
+
+const Link = ({ url }) => (
```

Comment:
Should we add this as Link-component under oskari-ui? With something like "external" boolean prop (default to true) to set target="_blank"?